@here/harp-vectortile-datasource/lib/VectorTileDataEmitter.ts

```diff
@@ -845,10 +845,9 @@ export class VectorTileDataEmitter {
 
             for (const polygon of geometry) {
                 const rings: Ring[] = [];
-
-                for (const outline of polygon.rings) {
-                    let coords = outline;
-
+                let discardPolygon: boolean = false;
+                for (let i = 0; i < polygon.rings.length; ) {
+                    let ringCoords = polygon.rings[i++];
```

Comment:
it was a left-over from a previous change where I needed the ring's index. However, in the context of the simplification of the nested check (as per the comment below), I left this notation so that it's possible to derive which ring is exterior from its index.